app/editor_ui/views/prompt_views.py

```diff
@@ -188,12 +201,19 @@ def get_context_data(self, **kwargs):
 
 
 class RangedPromptOptionsCreateView(
-    SuperuserRequiredMixin, LoginRequiredMixin, BaseCreateView
+    LoginRequiredMixin,
+    ProjectMembershipRequiredMixin,
+    BaseCreateView,
 ):
     model = RangedPrompt
```

Comment:
I don't think the `model` attribute needs to be specified here. Any reason to keep it?